google-cloud-monitoring/samples/Rakefile

```diff
@@ -18,3 +18,16 @@ Rake::TestTask.new "test" do |t|
   t.test_files = FileList["acceptance/*_test.rb"]
   t.warning = false
 end
+
+task "clean_fixtures" do
```

Comment:
I like `fixtures:clean` (using namespace) just in case we ever add any more fixtures tasks.

Reply:
In KMS:

```
namespace :fixtures do
  desc "Lists the existing KMS acceptance test fixtures for the project in us-east1"
  task :list do
```

Reply:
Changed to `fixtures:clean`.